pull_request_template.md

```diff
@@ -3,9 +3,13 @@ Write a detailed description of your PR here.
 Please follow the guidelines for PRs contained [here](docs/source/contributing.md). Checklist:
 
 - [ ] Ensure that your PR is small, and implements one change.
+- [ ] Give your PR title one of the prefixes FIX, ENH, STYLE, DOC, DEL to indicate what type of change that is.
+- [ ] Link the correct GitHub issue for tracking.
 - [ ] Add unit tests for all functions that you introduced or modified.
-- [ ] Run PyCharm's code cleanup tools on your Python files.
+- [ ] Run automatic code formatting / linting on all files.
 - [ ] Ensure that documentation renders correctly in Sphinx (run Sphinx via `make html` in the `docs folder)
```

Comment:
Any additional install requirements for this, or Sphinx is installed with the basic requirements?